modules/signalfx/test/java/com/spotify/ffwd/signalfx/SignalFxPluginSinkTest.java

```diff
@@ -0,0 +1,105 @@
+package com.spotify.ffwd.signalfx;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableSet;
+import com.signalfx.metrics.flush.AggregateMetricSender;
+import com.signalfx.metrics.protobuf.SignalFxProtocolBuffers.DataPoint;
+import com.signalfx.metrics.protobuf.SignalFxProtocolBuffers.Dimension;
+import com.spotify.ffwd.model.Metric;
+import eu.toolchain.async.AsyncFramework;
+import eu.toolchain.async.TinyAsync;
+import java.util.Collections;
+import java.util.Date;
+import java.util.List;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.function.Supplier;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+@RunWith(MockitoJUnitRunner.class)
+public class SignalFxPluginSinkTest {
```

Comment:
This should probably use failsafe since it's doing more complex things:
http://maven.apache.org/surefire/maven-failsafe-plugin/

Reply:
no need, this is not a real integration test, just using mocks